run.sh

```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=matomat
```

Comment:
add a
#!/bin/sh

to the first line

Reply:
Will remove the entire file, as it's a dev relic and not needed to start the program.